### ISSUE TITLE
Lightness Condition 

### ISSUE THREAD

**Issue by gisu** [ausi/cq-prolyfill#7]
Today i have tested your CQ Prolyfill, also the Lightness Check and i'm a little bit irritated because the Results - "its bright". With black and white Background anytime the same result. 

```css
.test {
  background-color: black; }

  .test__inner {
    color: #ccc; }

    .test:container("background-color lightness > 50%") .test__inner:before {
      content: 'its bright'; }
    .test:container("background-color lightness <= 49%") .test__inner:before {
      content: 'its dark'; }
```

I added a Test with a Single Condition with a Color (HSB Model with 30% Brightness)

```css
.test {
  background-color: #1a364d; }
  .test__inner {
    color: #ccc; }
    .test:container("background-color lightness > 50%") .test__inner:before {
      content: 'its bright'; }
```

Anytime "its bright". When i switch the Condition

```css
.test:container("background-color lightness < 50%") .test__inner:before {
      content: 'its bright'; }
```

I can set the Number from 0 to 100%, no results.

**Comment by ausi** [comment]
The container query is attached to the element you want to style. So instead of writing `.test:container("background-color lightness > 50%") .test__inner:before` like in other element query scripts, you append the query to the child itself `.test__inner:container("background-color lightness > 50%"):before`.

Your complete example should then look like:

```css
.test {
  background-color: black; }

  .test__inner {
    color: #ccc; }

    .test__inner:container("background-color lightness > 50%"):before {
      content: 'its bright'; }
    .test__inner:container("background-color lightness <= 49%"):before {
      content: 'its dark'; }
```

**Comment by gisu** [comment]
And here comes the Problem i got no results, the DOM is untouched from the Script. So no result here. With the Reference to the Parent:

```
.test:container("background-color lightness > 50%") .test__inner:before {
  content: 'its bright'; }
.test:container("background-color lightness <= 49%") .test__inner:before {
  content: 'its dark'; }
```
It works, its only crazy that the Result is 'its bright' with Background Color Black.

When i switch back to width like 

```
.test {}
.test__inner {}
.test__inner:container("width > 600px"):before {
  content: 'bigger than 600'; }
.test__inner:container("width < 600px"):before {
  content: 'smaller than 600'; }
```

Get also no Result. With a reference to a parent it works

```
.test:container("width > 600px") .test__inner:before {
  content: 'bigger than 600'; }
.test:container("width < 600px") .test__inner:before {
  content: 'smaller than 600'; }
```

**Comment by ausi** [comment]
There was a bug with pseudo elements (`:before`, `:after`) which should be fixed now. Could you please try if the latest version from the master branch works for you?

**Comment by gisu** [comment]
Tested with Width and Color - it works now. Thanks :)

**Comment by ausi** [comment]
Great. Thank you for reporting this bug!

I just released [version 0.2.5](https://github.com/ausi/cq-prolyfill/releases/tag/v0.2.5) which includes the fix.
